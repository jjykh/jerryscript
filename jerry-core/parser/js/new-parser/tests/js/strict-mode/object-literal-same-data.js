/* This test contains strict mode parser tests. */

'use strict'

a = {x:5,//
x:6};
