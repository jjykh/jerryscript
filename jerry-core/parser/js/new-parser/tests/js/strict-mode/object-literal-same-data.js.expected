
--- Script parsing start ---

  Note: switch to strict mode

  [  1] CBC_PUSH_LITERAL idx:0->string(use strict)
  [  0] CBC_POP_BLOCK
  [  1] CBC_CREATE_OBJECT
  [  2] CBC_PUSH_NUMBER_1 number:5

--- Script parsing end ---

Parse error 'Property of object literal redefined.' at line: 6 col: 1
